File: C#/백준/Silver/12723. Minimum Scalar Product （Small）/Minimum Scalar Product （Small）.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            for (int i=0; i<a; i++){
                int ans = 0;
                int b = int.Parse(Console.ReadLine());
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int[] bb = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                Array.Sort(aa);
                Array.Sort(bb);
                Array.Reverse(bb);
                for (int j=0;j<b;j++){
                    ans+=aa[j]*bb[j];
                }
                Console.WriteLine($"Case #{i+1}: {ans}");
            }
            
        }
    }
}

File: Python/백준/Silver/12723. Minimum Scalar Product （Small）/Minimum Scalar Product （Small）.py
a = int(input())
for i in range(a):
    ans = 0
    b = int(input())
    aa = list(map(int,input().split()))
    bb = list(map(int,input().split()))
    aa.sort()
    bb.sort(reverse="True")
    for j in range(b):
        ans+=aa[j]*bb[j]
    print(f"Case #{i+1}: {ans}")